recipe/meta.yaml

```
@@ -1,44 +1,70 @@
+{% set name = "uvicorn" %}
 {% set version = "0.16.0" %}
 
 package:
-  name: uvicorn
+  name: {{ name }}-recipe
```

I would explicitly write the name here instead of using Jinja variables.
```suggestion
  name: uvicorn-split
```